sass/edx/_variables.scss

```diff
@@ -175,238 +157,207 @@ $grid-breakpoints : (
-$border-radius-sm:       2px !default;  // .2rem !default;
-
-//$component-active-color: $white !default;
-//$component-active-bg:    $brand-primary !default;
-
-//$caret-width:            .3em !default;
-
-//$transition-base:        all .2s ease-in-out !default;
-//$transition-fade:        opacity .15s linear !default;
-//$transition-collapse:    height .35s ease !default;
-
-
+//$mark-bg: #fcf8e3 !default;
+//
+//
 // Tables
 //
 // Customizes the `.table` component with basic values, each used across all table variations.
-
+//
 //$table-cell-padding:            .75rem !default;
-//$table-sm-cell-padding:         .3rem !default;
+//$table-cell-padding-sm:         .3rem !default;
 //
 //$table-bg:                      transparent !default;
+//$table-accent-bg:               rgba($black,.05) !default;
+//$table-hover-bg:                rgba($black,.075) !default;
+//$table-active-bg:               $table-hover-bg !default;
 //
-//$table-inverse-bg:              $gray-dark !default;
-//$table-inverse-color:           $body-bg !default;
-//
-//$table-bg-accent:               rgba($black,.05) !default;
-//$table-bg-hover:                rgba($black,.075) !default;
-//$table-bg-active:               $table-bg-hover !default;
+//$table-border-width:            $border-width !default;
+//$table-border-color:            $gray-200 !default;
 //
-//$table-head-bg:                 $gray-lighter !default;
-//$table-head-color:              $gray !default;
+//$table-head-bg:                 $gray-200 !default;
+//$table-head-color:              $gray-700 !default;
 //
-//$table-border-width:            $border-width !default;
-//$table-border-color:            $gray-lighter !default;
+//$table-dark-bg:              $gray-900 !default;
+//$table-dark-accent-bg:       rgba($white, .05) !default;
+//$table-dark-hover-bg:        rgba($white, .075) !default;
+//$table-dark-border-color:    lighten($gray-900, 7.5%) !default;
+//$table-dark-color:           $body-bg !default;
 
 
 // Buttons
 //
 // For each of Bootstrap's buttons, define text, background and border color.
 
-$btn-padding-x:                  1rem !default;
-$btn-padding-y:                  .5rem !default;
-$btn-line-height:                1.25 !default;
+$input-btn-padding-y:       .5rem !default;
+$input-btn-padding-x:       .75rem !default;
+$input-btn-line-height:     1.25 !default;
+
+$input-btn-padding-y-sm:    .25rem !default;
+$input-btn-padding-x-sm:    .5rem !default;
+$input-btn-line-height-sm:  1.5 !default;
+
+$input-btn-padding-y-lg:    .5rem !default;
+$input-btn-padding-x-lg:    1rem !default;
+$input-btn-line-height-lg:  1.5 !default;
+
 $btn-font-weight:                $font-weight-normal !default;
-$btn-box-shadow:                 none !default;
-$btn-focus-box-shadow:           none !default;
-$btn-active-box-shadow:          none !default;
+$btn-box-shadow:                 inset 0 1px 0 rgba($white,.15), 0 1px 1px rgba($black,.075) !default;
```

Comment:
Can this use theme colors?

Reply:
I don't think we can nest functions, so theme-color("inverse") would not be correctly passed into the rgba. Instead, I'm going to make a separate $theme-color-inverse that we can pass in.

Reply:
Forget the above comment, turns out you can nest.